### DISCUSSION TITLE
Arrange images in a table

### DISCUSSION THREAD

**Discussion by aloboa** [zim-desktop-wiki/zim-desktop-wiki#2221] [Q&A]
Is it possible to include images within cells so that I can arrange images as grids?
If not, is there any other way to arrange images side by side?

**Comment by introt** [comment]
While the table editor plugin doesn't support images in table cells, you can have multiple images side by side by simply placing them on the same line. If the images share the same dimensions (and are resized as needed), multiple lines with multiple images will appear in a grid-like formation.

Do note that the "grid" will break if it doesn't fit horizontally in the app window. Afaik the export templates don't do any magic around image "grids" either. Another workaround without this issue is to pre-arrange your images in eg. GIMP and export them as one big image.

**Reply by introt** [reply]
Another another workaround is to use one of the image generator plugins which can layout images, eg. Insert Equation which uses LaTeX.

**Comment by aloboa** [comment]
Both workarounds are really inconvenient.
Markdown syntax does allow for including images within tables, eg
https://stackoverflow.com/questions/48667618/images-in-a-table-with-github-markdown
Why not in zim?

**Reply by introt** [reply]
A combination of "Zim is (currently) not a MarkDown editor" and "no one has implemented it".

**Comment by aloboa** [comment]
Then, this discussion should be re-converted to an issue, as a requested feature.

**Reply by introt** [reply]
There's an existing issue for this: #432
